packages/stack-javascript/eslint/jest.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  overrides: [
+    {
+      files: ["**/__tests__/**/*.spec.js"],
```

Comment:
@mvsde Should we use the default here that `eslint` uses to look for test files? See https://jestjs.io/docs/configuration#testmatch-arraystring.